models/nxos/route_maps/route_maps.yaml

```diff
@@ -0,0 +1,416 @@
+module: nxos_route_maps
+short_description: Route Maps resource module.
+description:
+- This module manages route maps configuration on devices running Cisco NX-OS.
+version_added: 2.0.0
+notes:
+- Tested against NX-OS 9.3.6.
+- This module works with connection C(network_cli) and C(httpapi).
+author: Nilashish Chakraborty (@NilashishC)
+options:
+  running_config:
+    description:
+    - This option is used only with state I(parsed).
+    - The value of this option should be the output received from the NX-OS device
+      by executing the command B(show running-config | section '^route-map').
+    - The state I(parsed) reads the configuration from C(running_config) option and
+      transforms it into Ansible structured data as per the resource module's argspec
+      and the value is then returned in the I(parsed) key within the result.
+    type: str
+  config:
+    description: A list of route-map configuration.
+    type: list
+    elements: dict
+    suboptions:
+      route_map:
+        description: Route-map name.
+        type: str
```

Comment:
There can be a list of entries (with sequence number as the primary key) with the same route_map_name. So I think there should be a list (route_maps) of dicts{ name: x , entries : [ {seqno: 10 ...}, {seqno: 20 ...} ...] }

Reply:
@GomathiselviS yes, you're right. I'll update the model. Thanks!